baufile.csx

```diff
@@ -0,0 +1,103 @@
+// parameters
+var versionSuffix = Environment.GetEnvironmentVariable("VERSION_SUFFIX");
+versionSuffix = string.IsNullOrWhiteSpace(versionSuffix) ? "-adhoc" : versionSuffix;
```

Comment:
It can be whatever you like, but this is the approach I take in other projects. On the build server, the VERSION_SUFFIX env var is set, and that is used for the build, so that can be set to alpha, beta, etc. depending on what state the default branch is at. The adhoc fallback here is for builds that are run outside the CI server, e.g. on your workstation, i.e. ad-hoc :wink:.

Now, with most projects, I don't fall back to anything here, I just leave it blank. But with this project, it depends on pre-release Roslyn packages, so it has to have *some* version suffix to make it a pre-release, otherwise NuGet refuses to package it, because it depends on pre-release packages.